### ISSUE TITLE
time guessing

### ISSUE THREAD

**Issue by snchzantonio** [manifestinteractive/teleprompter#13]
is it posible to Guess how long it takes to read the text according to the speed, the amount of text(lines), resolution and font size?

**Comment by manifestinteractive** [comment]
Yes, there are services that can figure out the Words Per Minute an average person can read and figure out how much time it would take to ready given text.  Though that is not something I believe should be in this project as the entire point of the timer is to tell you how long it is taking YOU to read the text you entered.  More often than not, a Teleprompter means there is a fixed time that text should be read in, so I don't honestly think it matters how long it SHOULD take to read.  I think what does matter is how long it DOES take to read, which are completely different functions.

Since this entire project allows the users to change the speed of their read, it would also be near impossible to both allow for a change in speed and also try to figure out on the fly how long something should take to read at that given speed.  There are just too many factors to consider with things like Font Size, Speed, Length of Text, etc.

Bottom line, if people need a solution to figure out how long a normal person would take to read a set of text, that will need to happen outside of this project for the reasons stated above.

**Comment by snchzantonio** [comment]
i see...good answer
